.travis.yml

```diff
@@ -33,13 +34,8 @@ before_install:
   - cat .bazelrc.travis .bazelrc.orig > .bazelrc
 
 script:
-  #- bazel --output_base=${HOME}/bazel/outbase test //...
   - bazel test //...
-  - bin/bazel_to_go.py
-  - ls -l ${TRAVIS_BUILD_DIR}
-  - go test -v ./pkg/... ./adapter/... ./cmd/...
```

Comment:
do we want to keep this line, or not?  i believe it was added to ensure `go test` and `bazel test` interop.